### ISSUE TITLE
Add libsmbclient's print-related functions

### ISSUE THREAD

**Issue by aklomp** [eduardok/libsmbclient-php#25]
Libsmbclient has various functions for printing to print shares, getting print queues and so on. We should add them, because it's easy to do so within the framework that we have and because it's the last large bit of upstream functionality we don't expose.

**Comment by eduardok** [comment]
Yes, I always hope to get back and add all the functions, but never had time. Will try starting next week.
